docs/gettingstarted/ansible-docker-swarm.md

```diff
@@ -1,23 +1,19 @@
-## Notice
-SC4S installation can now be automated with Ansible. All you need to do now is provide list of hosts on which you want to run SC4S and basic configuration (Splunk endpoint, HEC token, TLS configuration, etc.).
-This manual assumes that you have proper knowledge of Docker Swarm as setting up proper Swarm architecture/configuration is users duty.
+SC4S installation can be automated with Ansible. To do this, you provide a list of hosts on which you want to run SC4S and basic configuration, such as Splunk endpoint, HEC token, and TLS configuration. To perform this task, you must have existing understanding of Docker Swarm and be able to set up your Swarm architecture and configuration.
 
-# Initial Configuration
+# Step 1: Prepare your initial configuration
 
-All you need to do before running sc4s with Ansible is providing `env_file`. In the env file provide at least proper Splunk endpoint and HEC token.
-Create a file in `ansible/resources` catalog or edit [example file](/ansible/resources/env_file).
+1. Before running SC4S with Ansible, provide `env_file` with your Splunk endpoint and HEC token. To do this, create the file in `ansible/resources` catalog or edit [example file](/ansible/resources/env_file):
 
 ``` dotenv
 --8<---- "ansible/resources/env_file"
 ```
-Next provide a list of hosts on which you want to run Docker Swarm cluster and host application in inventory file:
+2. Provide a list of hosts on which you want to run your Docker Swarm cluster and the host application in the inventory file:
 ``` yaml
 --8<---- "ansible/inventory/inventory_swarm.yaml"
 ```
-You can run your cluster with one or more manager nodes for more info about setting up a swarm refer to [official docker documentation](https://docs.docker.com/engine/swarm/).
-NOTICE: One of biggest advantages of using Docker Swarm for hosting SC4S is Swarm internal load balancer (routing mesh). To get to know the details refer to [docker documentation](https://docs.docker.com/engine/swarm/ingress/).
+3. You can run your cluster with one or more manager nodes. One advantage of hosting SC4S with Docker Swarm is that you can leverage the Swarm internal load balancer. See your [Docker documentation](https://docs.docker.com). 
```

Comment:
can we preserve link to https://docs.docker.com/engine/swarm/ingress ?

Reply:
How about "See your Swarm Mode documentation at [Docker](https://docs.docker.com). " they should be able to find it since swarm mode is a major heading, and if the docs get moved or renamed, that will be enough info for them to google it (Our research shows that most users google our docs rather than looking through the manuals)

Reply:
all right :)